doc/source/features/strict-mode.rst

```diff
@@ -0,0 +1,384 @@
+.. _strict-mode:
```

Comment:
File name can be changed from 

doc/source/features/strict-mode.rst

doc/source/features/pxc-strict-mode.rst